sources/PPU/PPUUtils.hpp

```diff
@@ -17,11 +17,16 @@ namespace ComSquare::PPU
 
 namespace ComSquare::PPU::Utils
 {
-
-	//! @brief Transform SNES color code BGR to uint32_t RGB
-	uint32_t getRealColor(uint16_t color);
+	//! @brief Transform CGRAM color data to 8bits
+	//! @note Used with b, g and r values
+	//! @return The 8 bit value of the color
+	inline uint8_t to8Bit(int color);
```

Comment:
I believe inlined functions must be in the header file. I might be wrong tho.

Reply:
i looked it's doable but not what  I want